[2023-04-25T08:29:56.516+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: database_processing.create_table manual__2023-04-25T08:29:54.941175+00:00 [queued]>
[2023-04-25T08:29:56.525+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: database_processing.create_table manual__2023-04-25T08:29:54.941175+00:00 [queued]>
[2023-04-25T08:29:56.526+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-04-25T08:29:56.526+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-04-25T08:29:56.527+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-04-25T08:29:56.541+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): create_table> on 2023-04-25 08:29:54.941175+00:00
[2023-04-25T08:29:56.547+0000] {standard_task_runner.py:55} INFO - Started process 9045 to run task
[2023-04-25T08:29:56.554+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'database_processing', 'create_table', 'manual__2023-04-25T08:29:54.941175+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/database_processing.py', '--cfg-path', '/tmp/tmp6d0gvtjg']
[2023-04-25T08:29:56.555+0000] {standard_task_runner.py:83} INFO - Job 13: Subtask create_table
[2023-04-25T08:29:56.626+0000] {task_command.py:388} INFO - Running <TaskInstance: database_processing.create_table manual__2023-04-25T08:29:54.941175+00:00 [running]> on host 2f9666fcabde
[2023-04-25T08:29:56.699+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=database_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-04-25T08:29:54.941175+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-25T08:29:54.941175+00:00
[2023-04-25T08:29:56.700+0000] {sql.py:254} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS USERS (
                id INT,
                name VARCHAR(255) NOT NULL,
                email VARCHAR(255) NOT NULL,
                phone VARCHAR(255) NOT NULL,
                address VARCHAR(255) NOT NULL,
                organization VARCHAR(255) NOT NULL
            );
        
[2023-04-25T08:29:56.722+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-04-25T08:29:56.859+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-04-25T08:29:56.867+0000] {sql.py:375} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS USERS (
                id INT,
                name VARCHAR(255) NOT NULL,
                email VARCHAR(255) NOT NULL,
                phone VARCHAR(255) NOT NULL,
                address VARCHAR(255) NOT NULL,
                organization VARCHAR(255) NOT NULL
            );
        , parameters: None
[2023-04-25T08:29:56.886+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=database_processing, task_id=create_table, execution_date=20230425T082954, start_date=20230425T082956, end_date=20230425T082956
[2023-04-25T08:29:56.924+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-04-25T08:29:56.948+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
